reV/bespoke/gradient_free.py

```diff
@@ -49,9 +49,19 @@ def __init__(self, bits, bounds, variable_type, objective_function,
         convergence_iters : int, optional
             The number of generations to determine convergence.
         max_time : float
-            The maximium time (in seconds) to run the genetic algorithm.
+            The maximum time (in seconds) to run the genetic algorithm.
         """
 
+        logger.info('Initializing GeneticAlgorithm...')
+        logger.info('Minimum convergence iterations: {}'
+                    .format(convergence_iters))
+        logger.info('Max iterations (generations): {}'.format(max_generation))
+        logger.info('Population size: {}'.format(population_size))
+        logger.info('Crossover rate: {}'.format(crossover_rate))
+        logger.info('Mutation rate: {}'.format(mutation_rate))
+        logger.info('Convergence tolerance: {}'.format(tol))
+        logger.info('Maximum runtime (in seconds): {}'.format(max_time))
```

Comment:
Can you just verify that these dont get parallel logged when doing multi-node multi-core jobs? I try to keep the parallelized logging statements as debug and the serial/manager level as info.

Reply:
Oops, I think they do get parallel logged. I will keep that in mind moving forward. Let me make adjustments!